frontend/src/components/core/AppView/AppView.tsx

```diff
@@ -145,12 +147,16 @@ function AppView(props: AppViewProps): ReactElement {
       )}
       <StyledAppViewMain tabIndex={0} isEmbedded={embedded} className="main">
         {renderBlock(elements.main)}
-        <StyledAppViewFooter isEmbedded={embedded} isWideMode={wideMode}>
-          Made with{" "}
-          <StyledAppViewFooterLink href="//streamlit.io" target="_blank">
-            Streamlit
-          </StyledAppViewFooterLink>
-        </StyledAppViewFooter>
+        <StyledIFrameResizerAnchor isEmbedded={embedded} data-iframe-height />
+        {!embedded && <StyledAppViewBlockSpacer />}
```

Comment:
Instead of the block stretching to fit, we have a spacer that will fill remaining space if necessary

Reply:
Worth an inline comment in the code?